### PR TITLE
Support both yaml and yml extensions for package and metadata

### PR DESCRIPTION
**Pull request by frozzare**
Support both yaml and yml extensions when reading, not writing (e.g build command)

### CODE REVIEW

File: internal/pkg/package.go [hunk 1]
```diff
@@ -221,7 +222,15 @@ func GetName(importPath string, isSrc bool) string {
 // Metadata returns the package metadata
 func (p *file) Metadata() (Meta, error) {
 	var m Meta
-	if err := yaml.Unmarshal(p.content["package.yaml"], &m); err != nil {
+	var c []byte
+
+	if len(p.content["package.yaml"]) > 0 {
```
Comment (creekorful) [inline internal/pkg/package.go:227]: if ok, val := p.content["package.yaml"]; ok {
   ...
}

**Review by creekorful — COMMENTED** [review]
Little implementation suggestion, otherwise lgtme :)

File: internal/pkg/package.go (hunk 1 above)
Comment (frozzare) [reply internal/pkg/package.go:227]: Ah, didn't know that you could that on map/bytes, I will fix!

**Review by creekorful — APPROVED** [review]

**Review by Johannestegner — APPROVED** [review]
LGTM :)

**Pull request merged**